### ISSUE TITLE
Do we need to use <lb/> at the beginning of an <l> element?

### ISSUE THREAD

**Issue by scottkleinman** [scottkleinman/aeme#24]
That is: <l><lb/>Beginning of the line...</l>

@skgoetz: 
I wouldn’t, as you know…. See first example at http://www.tei-c.org/release/doc/tei-p5-doc/en/html/CO.html#COVE

I'm not convinced by this example, the logic of which is described thus: "By convention, the start of a metrical line implies the start of a typographic line". Of course, we are not dealing with typographic lines, but, as an Anglo-Saxonist, I am used to the start of a metrical line _not_ implying the start of a line in the manuscript.

This is not to say that we have to include <lb/> at the beginning of each metrical line where it _does_ correspond to a new line in the manuscript. It just means that we have to be able to render both types of text in the diplomatic layer. Is a stylesheet up to the task?

**Comment by scottkleinman** [comment]
It's time to follow up on this thread. Laud Misc. 108, ff. 60r-60v (SEL life of St Lucy) switches from writing out verse with line breaks to writing out verse as prose in the middle of a the page. In order not to suggest that the poem has structural divisions, I have used `<lg rend="prose">` rather than `<divX>`. But this should be reconsidered down the line.

**Comment by dorothyk98** [comment]
I think that's fine. Just one question. What happens down the line if someone wants to scoop up data wise the bits across the archive that are "prose"? Would the rend itself allow the search or what would work best? I do think this will come up again.

**Comment by skgoetz** [comment]
Either way of tagging it ought to let someone winnow, as long as it's done consistently. 

I rather object to the notion of a prose line group, but since I am receding into the distance, perhaps it doesn't matter.
--
Sent via phone.

**Comment by scottkleinman** [comment]
Yes, as long as you can query attributes, it doesn't matter which attribute you use. I take the point about the oxymoronic prose line group, although, if you use `@rend` it technically means that the line group was "rendered or presented in the source text" as prose. This may be slightly more palatable. But I also question my original fear of using `<div>`, so this is something we can reconsider. It only occurs in a couple of places in Laud Misc. 108, so we'll see if Junius 1 provides scenarios that give us more insight into treatment of this phenomenon.

**Comment by scottkleinman** [comment]
More fun with lineation. The Laud Misc. 108 *Debate between the Body and Soul* is written out with one manuscript line consisting of two verse lines per manuscript line (see f. 200v). It *looks* like each line consists of two half-lines, but the rhyme scheme clearly indicates that the two half-lines are separate lines (as an annotator has marked by drawing lines between the connecting b-verses in the first stanza). How to capture this? None of the recommendations in the [TEI Guidelines](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/VE.html#VESE) seem much use for this kind of structure, apart from nested `<lg>` elements, which would be tedious and verbose. The following seems to me to capture some of what is happening:

```xml
<lg rend="prose">
  <l xml:id="bodllaudmisc108.6.1" rhyme="A">Wan þe gost it scholde go<pc>&punctelev;</pc></l>
  <l xml:id="bodllaudmisc108.6.2" rhyme="B">yt biwente and withstod</l>
  <l xml:id="bodllaudmisc108.6.3" rhyme="A">Biheold the body þere it cam fro<pc>&punctelev;</pc></l>
  <l xml:id="bodllaudmisc108.6.4" rhyme="B">so serfulli with dredli mod</l>
  <l xml:id="bodllaudmisc108.6.5" rhyme="A">It seide weile and walawo<pc>&punctelev;</pc></l>
  <l xml:id="bodllaudmisc108.6.6" rhyme="B">Wo worþe þi fleys þi foule blod</l>
  <l xml:id="bodllaudmisc108.6.7" rhyme="A">Wreche bodi wȝy listouȝ so<pc>&punctelev;</pc></l>
  <l xml:id="bodllaudmisc108.6.8" rhyme="B">þat ȝwilene were so wilde and wod</l>
</lg>
```
That is, eight lines, rhyming ABABABAB, with each A verse ending in a *punctus elevatus*. However, we can't render this as "prose" because the manuscript contains a line break on each A-verse. We *could* add `<lb/>`, but this leaves open how we render the code in HTML. It would be simplest to transform the above code to HTML as follows:
```html
<div class="prose">
  <p class="A">Wan þe gost it scholde go;</p>
  <p class="B">yt biwente and withstod</p>
  <p class="A">Biheold the body þere it cam fro;</p>
  <p class="B">so serfulli with dredli mod</p>
  <p class="A">It seide weile and walawo;</p>
  <p class="B">Wo worþe þi fleys þi foule blod</p>
  <p class="A">Wreche bodi wȝy listouȝ so;</p>
  <p class="B">þat ȝwilene were so wilde and wod</p>
</div>
```
Give it the following CSS:
```css
p {margin:0;}
.prose p {display: inline;}
.prose p.B:after {content: '\A'; white-space: pre;}
```
This will correctly render the diplomatic text as:

Wan þe gost it scholde go; yt biwente and withstod
Biheold the body þere it cam fro; So serfulli with dredli mod
It seide weile and walawo; Wo worþe þi fleys þi foule blod
Wreche bodi wȝy listouȝ so; þat ȝwilene were so wilde and wod
 
And turning off the prose class will render the text with line breaks for every paragraph in the critical view.

So I'm just throwing this out to see if anyone has comments or other ideas. Would such minimal markup break if CSS were to change radically? (I suppose it would still be possible to generate useable markup programmatically; I'm just trying to avoid that with this solution.)

**Comment by mmwwah** [comment]
No argument from me.  And yeah, if CSS changes radically, we should be able to script a revision in our markup, I should think, so I don't see that as a problem.

**Comment by scottkleinman** [comment]
I should add that the proposed solution is not particularly semantic--the text is not prose. We could use another value for `@rend` with the same CSS. But what to call it? Half-line couplets? Note that you could interpret the *Ormulum* as having a similar verse form, but with "lines" designated by syllable count instead of rhyme and without the graphic line break after the b-verse.

**Comment by skgoetz** [comment]
What in CSS would you expect to break or be concerned about breaking? By then it's more likely that something would have superseded CSS entirely.

"halfline" would make sense to me if you went that route.
